### PR TITLE
Better handling of uninitialized primitives

### LINKED ISSUE #809: Not use MaybeUninit for primitive types
It's a bit overkill to use it on primitives considering we could just initialize them like `let x = 0;`. Less code for the same result.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #809.

cc @EPashkin @sdroege

### CODE REVIEW

File: src/library.rs [hunk 1]
```diff
@@ -250,6 +250,32 @@ impl Fundamental {
...
+            | Fundamental::Char
+            | Fundamental::UChar
+            | Fundamental::Short
+            | Fundamental::UShort
+            | Fundamental::Int
+            | Fundamental::UInt
+            | Fundamental::Long
+            | Fundamental::ULong
+            | Fundamental::Size
+            | Fundamental::SSize => Some("0"),
```
Comment (sdroege) [inline src/library.rs]: `let b: bool = 0` is not going to work :)

Comment (GuillaumeGomez) [reply src/library.rs]: Is there any boolean declared anywhere?

**Comment by sdroege** [comment]
I'm not entirely convinced that this is the right thing to do btw, I only mentioned it on IRC because changing everything manually is quite some work. For autogenerated code, using `MaybeUninit` has no additional cost to humans and is minimally faster for computers.

File: src/library.rs (hunk 1 above)
Comment (sdroege) [reply src/library.rs]: 15 lines above or so in the same block of code

Comment (EPashkin) [reply src/library.rs]: At minimum no booleans in gtk-rs

**Comment by EPashkin** [comment]
Sorry, @GuillaumeGomez, I also think that `MaybeUninit` is better for this case, at minimum in generated code.

PS. If you still insist, then this PR breaks atk and clippy found unneeded `clone` for `&'static str`.

**Comment by GuillaumeGomez** [comment]
No, @sdroege convinced me so I'll just close it. :)

**Pull request closed without merging**